holoviews/plotting/bokeh/util.py

```diff
@@ -125,7 +128,7 @@ def mpl_to_bokeh(properties):
     return new_properties
 
 
-def layout_padding(plots):
+def layout_padding(plots, renderer):
     """
     Temporary workaround to allow empty plots in a
     row of a bokeh GridPlot type. Should be removed
```

Comment:
The PR reference in this docstring https://github.com/bokeh/bokeh/issues/2891 looks like it was resolved to your satisfaction in June. This suggests either this function should be removed or the docstring updated.